mapper/thin_edge_json/src/parser.rs

```diff
@@ -39,7 +39,7 @@ pub struct ThinEdgeJsonParserError {
 
 /// Parses top-level ThinEdge JSON:
 ///
-/// ```ignore
+/// ```grammar
```

Comment:
This is a fake keyword!

* The ignore keyword was interpreted by `cargo`.
* I picked this one because the block is actually a grammar description.
* If misleading, one can simply put nothing!